src/router.tsx

```diff
@@ -25,7 +24,7 @@ class RouterApp extends Component<RouterAppProps> {
   public render(): ReactElement<HTMLDivElement> {
     return (
       <Router history={history}>
-        <>
+        <React.Suspense fallback={null}>
```

Comment:
Please add the loading component here
```suggestion
        <React.Suspense fallback={<Loading />}>
```

Reply:
For now the `asyncComponent` renders `null` so I wanted to keep the existing behavior. But I can change it to display a loading component

Reply:
`<Loading/>` already are defined in each page, we can skip this.